setup.py

```diff
@@ -24,6 +24,7 @@
 ]
 
 EXTRAS_REQUIRE = {
+    'ci': ['teamcity-messages'],
```

Comment:
Is this used somewhere (e.g. `pip install .[ci]`) in the Docker build/Teamcity tests?

Reply:
yes exactly https://ene-builds.iiasa.ac.at/admin/editRunType.html?id=buildType:IxmpCore_IxmpPython&runnerId=RUNNER_7&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253AIxmpCore_IxmpPython%26init%3D1&cameFromTitle=